steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
I think I understand your approach now, but like I indicated before, this influences the other places where load_json_files is used, but in a less obvious way and maybe even in a way that is wanted. E.g. `steve-cmd status` currently gives lines like:

    json/0001_some_great_talk_on_python.json     Done!

and in my tries before my tristate parameter I would return the full path, making `steve-cmd status` return lines with the full path (making lines much longer than 80 characters etc). With your changes I think you will just lose the initial "json/", which might sound great. But other tools that expect the list from `steve-cmd status --list` exepct the "json/" to be there (and to be run from the project directory itself).
 I don't think it is a good idea to have those side effects, as I don't understand what other tools rely on previous behaviour well enough (read not at all) to make that decision.